data/human/variants.txt

```diff
@@ -81,6 +81,45 @@ ship "Argosy" "Argosy (Turret)"
 		"Hyperdrive"
 
 
+ship "Argosy" "Argosy (Plasma Repeater)"
+	outfits
+		"Anti-Missile Turret"
+		"D23-QP Shield Generator"
+		"Greyhound Plasma Thruster"
+		Hyperdrive
+		"Impala Plasma Steering"
+		"LP072a Battery Pack"
+		"Liquid Nitrogen Cooler"
+		"Outfits Expansion"
+		"Plasma Repeater" 2
+		"RT-I Radiothermal"
+
+
+ship "Argosy" "Argosy (Mega Blaster)"
```

Comment:
Not too sure about variant name.

Reply:
To be honest I don't know either. I could do "Argosy (Eight Blasters)"?

Reply:
(Twin Blaster)? (Blaster II)?

Reply:
It's now "Large Blaster".

Reply:
I think it shouldn't imply any kind of size to avoid potential conflict in the future if we ever want to have larger blasters or i plugins add them(pretty sure some existing one already did).

Reply:
Fair enough, I did not think of that.